src/test/scala/edu/ie3/simona/event/listener/ResultEventListenerSpec.scala

```diff
@@ -402,6 +402,9 @@ class ResultEventListenerSpec
           max = timeoutDuration,
         )
 
+        // Debug: Check if the file exists
```

Comment:
Do we need this?

Reply:
@PhilippSchmelter added while fixing the test. I can remove the comment or change it, if you think it is necessary or better.

Reply:
I will remove them with the next commit.